R/wrangle_and_check.R

```diff
@@ -126,3 +125,24 @@ wrangle_financial_data <- function(financial_data, start_year) {
 
   return(financial_data)
 }
+
+#' Wrangle scenario data
+#'
+#' Function applies custom wrangling to scenario data.
+#'
+#' @param scenario_data A tibble holding scenario data.
+#' @param start_year Start year of analysis.
+#' @param end_year End year of analysis.
+#'
+#' @return A tibble holding wrangled scenario_data.
+wrangle_scenario_data <- function(scenario_data, start_year, end_year) {
+  scenario_data_wrangled <- scenario_data %>%
+    dplyr::rename(source = .data$scenario_source) %>%
+    dplyr::filter(.data$source %in% c("ETP2017", "WEO2019")) %>%
```

Comment:
we need to remember to parameterise these scenarios at some point

Reply:
we sure do